Source/Immutable/Public/Immutable/ImmutablePassport.h

```diff
@@ -52,14 +52,36 @@ class IMMUTABLE_API UImmutablePassport : public UObject
 public:
 	DECLARE_MULTICAST_DELEGATE(FOnPassportReadyDelegate);
 
+	/**
+	 * Delegate used for JavaScript callbacks.
+	 */
 	DECLARE_DELEGATE_OneParam(FImtblPassportResponseDelegate, FImmutablePassportResult);
 
+	/**
+	 * Initialises passport. This sets up the Passport instance, configures the web browser, and waits for the ready signal.
+	 * @param InitData Parameters to initialise the passport with.
+	 * @param ResponseDelegate Callback delegate.
+	 */
 	void Initialize(const FImmutablePassportInitData& InitData, const FImtblPassportResponseDelegate& ResponseDelegate);
+
+	/**
+	 * Logs the user into Passport via device code auth and sets up the Immutable X provider.
+	 * This will open the user's default browser and take them through Passport login. 
+	 * @param IsConnectImx If true, attempts to re-connect using saved credentials. Else, attempts to re-login using saved credentials.
```

Comment:
```suggestion
	 * @param IsConnectImx If true, the "re-connect" method is used to authenticate into Passport with Immutable X. Otherwise, "re-login" is used for authentication. To access a wallet with Immutable X or zkEVM later, you must call "Connect" again with this value set to true, or use "ConnectEvm."
```